### ISSUE TITLE
Can i use batchnormalize to replace instancenormalize

### ISSUE THREAD

**Issue by Arthur8312** [yl4579/StarGANv2-VC#87]
I want to try to put the Generator into edge device.
But the instance normalize can not fusing into Conv
I have try replace batch normalize to replace instance normalize
or open instance normalize track_running_stats
The final result cannot produce normal sound. 
I'm wonder why is that. Or any suggestion?

**Comment by yl4579** [comment]
No, only instance normalization removes the speaker features. Batch normalization does not normalize per channel, so you can't do it that way. You may want to implement your own instance normalization simply by computing the mean and variance per channel.
